mautic/mautic
# Mautic Upgrade Prep Command

Adds mautic:prepare:upgrade command to capture all local files that will be required for a fresh install of Mautic 2.0

**Testing**

- Apply PR
- Run `php app/console mautic:prepare:upgrade` command.
- Check for creation of `mautic.zip` in your root folder
- Unzip mautic.zip and it should contain a copy of everything in your media, plugins, themes and translations folder as well as `app/config/local.php` and `app/config/*_local.php` if you have those.
